eng/source-build-patches/0001-Remove-test-and-sample-projects-from-solution.patch

```diff
@@ -0,0 +1,63 @@
+From fc9b0cac419453b28678689d670e4fc6e09fa143 Mon Sep 17 00:00:00 2001
```

Comment:
Should remove this file?

Reply:
Per the [Arcade Powered Source-Build plan](https://github.com/dotnet/source-build/blob/master/Documentation/planning/arcade-powered-source-build/implementation-plan.md), patch removal is something the product/repo teams will be responsible for removing after the initial infrastructure is checked in.  We would like to get phase 1 complete to unblock other work.